front/app/containers/Admin/projects/project/events/components/DateTimeSelection/TimeInput.tsx

```diff
@@ -77,7 +86,7 @@ const TimeInput = ({ selectedDate, onChange }: Props) => {
               bgHoverColor={i === timeIndex ? colors.teal700 : colors.teal100}
               onClick={() => handleButtonClick(i)}
             >
-              {time}
+              {format(time, 'p', { locale: getLocale(locale) })}
```

Comment:
We're now formatting this correctly based on your locale (e.g. 1:30 PM for English locales and 13:00, copied this behavior from the code of the other library)